chapters/spec-050-clus-class-exercise/exercise.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Exercises}
+
+\section{Clustering}
+
+Open the “Liver-cirrhosis” data set (\widget{Datasets} widget). It represents an a sliced liver tissue. It was sliced so thin that some parts of the image have very low intensities; let's call these "background".
```

Comment:
There is a typo here.

Also, the "background" regions originate from the fact that there are actually holes in the tissue, not that it is very thinly sliced.

I would say:

Open the “Liver-cirrhosis” data set from the (\widget{Datasets} widget). It represents a sliced liver tissue. As tissues have holes and they also tear during the slicing some parts of the dataset has very low spectral intensities. Let's call these "background regions".